.devcontainer/README.md

```
@@ -44,20 +44,20 @@ If you need to test with a LibreNMS instance on a private network (local lab, co
 
 1. Fork and Clone: fork the plugin repo in Github and clone locally
 2. Open in VS Code and choose "Reopen in Container" (or Ctrl+Shift+P → Dev Containers: Reopen in Container)
-2. Wait for setup (~5min on first run or when new NetBox image is used). The container will install the plugin and prep NetBox
-3. Set up GitHub access: `gh auth login` (for pushing/pulling code changes)
-4. Create your plugin config — see [Plugin configuration](#plugin-configuration):
+3. Wait for setup (~5min on first run or when new NetBox image is used). The container will install the plugin and prep NetBox
+4. Set up GitHub access: `gh auth login` (for pushing/pulling code changes)
+5. Create your plugin config — see [LibreNMS Server configuration](#librenms-server-configuration):
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)